lib/components/mp4muxer/index.ts

```diff
@@ -114,6 +166,38 @@ export class Mp4Muxer extends Tube {
       },
     })
 
+    const sendFtypMoovFromSdpMessage = (msg: SdpMessage): any => {
+      // Why is this here? These should be default inside the mvhd box?
```

Comment:
should this be made into a todo type message to investigate at some point?

Reply:
This is moved code!